### ISSUE TITLE
Incorrect parsing of floating point values

### ISSUE THREAD

**Issue by gdmsl** [JuliaData/CSV.jl#265]
Hi,
using  `CSV.read("filename")` or `CSV.read("filename", types=[Float64, Float64])` on this file

```
r,G
2.000000000000000000e+01,9.670879999999999537e-04
```
results in bad parsing on the second column:

```
r | G
-- | --
20.0 | -0.000877586
```

I checked that  `parse("9.670879999999999537e-04")` is working properly (and it is).

**Comment by quinnj** [comment]
This was fixed with the Parsers v0.2.2 release in 0.7/1.0. Are you on julia 0.6? Otherwise, you may need to upgrade that dependency.

**Comment by gdmsl** [comment]
Yes I am on 1.0. You are right. It works with the version in master but installing csv as
```
(v1.0) pkg> add CSV
```
will install CSV v0.3.1 (for which parsing doesn't work), while
```
(v1.0) pkg> add CSV#master
```
will install CSV v0.2.5#master.
Probably there are some mistakes in the Project.toml or in the Manifest.toml files. Also I noticed that installing the version from master downgrades RDatasets from v0.5.0 to v0.4.0. I don't know if this behavior is intended.
